src/Externs.cs

```diff
@@ -64,7 +64,11 @@ public partial class __default {
     }
 
     private static Double BigRationalToDouble(BigRational r) {
-      return (double)r.num / (double)r.den;
+      if (r.den == 0) {
+        return 0.0;
```

Comment:
I find it just terrifically ironic that that we managed to introduce division by zero here. :) I'm so used to Dafny now that I missed it in review myself!

Reply:
This is, unfortunately, a bug in the new `DafnyRuntime` code. And, actually, we can detect it by checking the numerator, instead, which is more mathematically sensible, so I'm going to change it to that (along with fixing the `DafnyRuntime` bug).

And, as to terminology, I like `--max-resource-cv` if it's a common abbreviation (and it also looks to me like it is).

Reply:
Ah, the Dafny runtime issue is hard to avoid, as Rustan [pointed out](https://github.com/dafny-lang/dafny/blob/master/Source/DafnyRuntime/DafnyRuntime.cs#L1499). But I think checking `num` is still a more mathematically sensible way of detecting the situation, so I've done that.